### PR TITLE
[UR][CUDA] Remove unused function variant

### PR DESCRIPTION
**Pull request by pasaulais**
Run CI for https://github.com/oneapi-src/unified-runtime/pull/1310

### CODE REVIEW

**Review by kbenzie — APPROVED** [review]

**Comment by kbenzie** [comment]
@intel/sycl-graphs-reviewers please review

File: sycl/test-e2e/Graph/Threading/submit.cpp
Comment (EwanC) [inline sycl/test-e2e/Graph/Threading/submit.cpp:1]: I'm not sure why this change from merged in https://github.com/intel/llvm/pull/12777 is included in this PR

Comment (kbenzie) [reply sycl/test-e2e/Graph/Threading/submit.cpp:1]: Okay, I'll see if something weird has gone on with updating this branch.

Comment (kbenzie) [reply sycl/test-e2e/Graph/Threading/submit.cpp:1]: Rebase fixed it. Thanks for pointing out this wasn't supposed to be in here.

**Comment by kbenzie** [comment]
@intel/llvm-gatekeepers please merge
